packages/host/app/components/operator-mode/card-url-bar.gts

```diff
@@ -159,6 +159,7 @@ export default class CardURLBar extends Component<Signature> {
         outline: none;
         padding: 0;
         min-height: 0;
+        text-overflow: ellipsis;
```

Comment:
Usually this is accompanied with `overflow: hidden; white-space: nowrap;` but maybe because this is an input field, it already has those properties 🤔

Reply:
ya I was surprised, I couldn’t find why it wasn’t needed but I figured the minimum change was best!